### ISSUE TITLE
Az Configure - Changeable settings aren't listed.

### ISSUE THREAD

**Issue by bgklein** [Azure/azure-cli#8209]
**Is your feature request related to a problem? Please describe.**
When 'az configure' is ran, it lists account details for configured through the CLI.
![image](https://user-images.githubusercontent.com/18502617/50916361-d26d4a00-13ef-11e9-8793-0c93d4706edd.png)
Then it asks if you want to change your settings.
![image](https://user-images.githubusercontent.com/18502617/50916382-e2852980-13ef-11e9-836a-e8c837864668.png)
And then lists a bunch of configurations which weren't displayed.
![image](https://user-images.githubusercontent.com/18502617/50916443-ffb9f800-13ef-11e9-893e-4a746fcbb87e.png)

**Describe the solution you'd like**
Either allow account details to be manually changed or add the configurable settings to the initial display.

**Additional context**
Basically if one were to say I want to change config settings they would go to az configure and see the current set of configurations.
Do you wish to change your settings then implies do you want to change the settings I just told you.

**Comment by bgklein** [comment]
2.0.44 I'll update and check again

**Comment by bgklein** [comment]
Yes and no(What you suggested seems appropriate as well). On a fresh 2.0.54 CLI my output is:
![image](https://user-images.githubusercontent.com/18502617/51094985-249eca00-1766-11e9-98d2-1fec083e41b5.png)

I mainly wanted to point out the fact that the only 'configurable' settings in this menu, are not being defaulted/being set.

For the workflow where the issue will occur:
    Problem: I want to change my output format.
    Current User Behavior: Go to az configure => See there isn't a defined option for output => Prompted 'Do you want to change anything' => User never realizes hidden setting

**Comment by bgklein** [comment]
@tjprescott any status update on this? doesn't necesarily need to be high pri if their are core CLI work items, but an approximate timeline may be nice?

**Comment by tjprescott** [comment]
@yugangw-msft @achandmsft for comment.

**Comment by yonzhan** [comment]
add to S164.
